From: Ivan Molodetskikh <yalterz@gmail.com>
Date: Sun, 6 Mar 2022 15:03:29 +0300
Subject: windowManager: Change unfullscreen mode to NORMAL

The unfullscreen action doesn't make sense in other modes. In the
overview in particular it also prevents clicking on the top bar items on
a workspace with a fullscreen client.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2227>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/4bc195e7
---
 js/ui/windowManager.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/windowManager.js b/js/ui/windowManager.js
index cdd32dc..bd47e6e 100644
--- a/js/ui/windowManager.js
+++ b/js/ui/windowManager.js
@@ -941,7 +941,7 @@ var WindowManager = class {
         appSwitchAction.connect('activated', this._switchApp.bind(this));
         global.stage.add_action_full('app-switch', Clutter.EventPhase.CAPTURE, appSwitchAction);
 
-        let mode = Shell.ActionMode.ALL & ~Shell.ActionMode.LOCK_SCREEN;
+        let mode = Shell.ActionMode.NORMAL;
         let topDragAction = new EdgeDragAction.EdgeDragAction(St.Side.TOP, mode);
         topDragAction.connect('activated',  () => {
             let currentWindow = global.display.focus_window;
